Fixes to <head> to fix console errors.

kevinjcoleman opened the pull request:
Removed an extra style tag in the <head> of tate's site. I'm going to be throwing a bunch more PR's your way, just wanted to see if I could merge with ya.

Fixed up font awesome references you had the file structure a little messed up.

kevinjcoleman committed:
Removed extra <style>

kevinjcoleman committed:
Fixed font awesome references

kevinjcoleman committed:
Changes to queproblema.html, made the loading screen a little better and set the width of type to 50% instead of pixels.

alexG53090 commented:
Kevin!! OMG, so happy that you're picking this up! Please feel free to just tear it up! Let me know if anything catastrophic happens. I don't update the live site via git, I just throw the files up in go daddy. :)) <3 sorry ive been awol. School is so busy -- been programming arduino robots! Also, tate, so stoked to see you diving right in as well. This is SO awesome. Keep up the good work!

kevinjcoleman commented:
Bennett!! So stoked to help out, I know you're super busy. I won't tear it
up, the structure is still super good! They were just a couple errors here
and there that I cleaned up, and then Tate and I are going to put a new
song in soon.

So you don't SSH into GoDaddy? You literally just go to the File Manager
and upload the files one at a time? Urghhh that's annoying, I wish the
changes to the master branch in Git could just be pushed automatically, it
would definitely make life easier.

Dude that's so epic! What language are you using to program arduino robots?
Send me a snapchat of them moving around I just added you hahah



On Wed, Oct 28, 2015 at 9:16 AM, Alex <notifications@github.com> wrote:

> Kevin!! OMG, so happy that you're picking this up! Please feel free to
> just tear it up! Let me know if anything catastrophic happens. I don't
> update the live site via git, I just throw the files up in go daddy. :)) <3
> sorry ive been awol. School is so busy -- been programming arduino robots!
> Also, tate, so stoked to see you diving right in as well. This is SO
> awesome. Keep up the good work!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alexG53090/tateTucker/pull/10#issuecomment-151897360>.
>



-- 
Kevin James Coleman
NationBuilder
<http://t.sidekickopen23.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJN7t5XZscVPq2W5wvtGC4WYnGHW8q-c7s56dzkrf7WCymb02?t=http%3A%2F%2Fwww.nationbuilder.com%2F&si=5000493221281792&pi=07a1fe88-bbcd-4e68-c2f3-a465c30c6e8b>
 Organizer
714-580-2061
